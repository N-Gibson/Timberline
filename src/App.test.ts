import React from 'react'
// import { shallow } from 'enzyme'
// import App from './App'

// describe('App', () => {
//   const wrapper = shallow(<App />)

//   it('App component should match the snapshot', () => {
//     expect(wrapper).toMatchSnapshot()
//   })
// })

// Running into ts errors when trying to make a shallow render of App
// Could be problem in the tsconfig
