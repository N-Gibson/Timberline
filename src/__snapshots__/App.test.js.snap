// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App component should match the snapshot 1`] = `
<main
  className="App"
>
  <Nav
    active="home"
  />
  <Route
    exact={true}
    path="/"
    render={[Function]}
  />
  <Footer />
</main>
`;
